code/modules/admin/tabs/event_tab.dm

```diff
@@ -897,7 +894,12 @@
 		if(alert("Are you SURE you want to do this? It will create an OB explosion!",, "Yes", "No") != "Yes") return
```

Comment:
Since the qdel is removed from above, we ought to be sure the warhead is deleted if the admin changes their mind. Make sure it is either qdeleted or has QDEL_IN set for all scenarios.